docs/02-Install.md

```diff
@@ -1,13 +1,111 @@
 # 2.0 - Install
 
-## 2.1 Dependencies
+## 2.1 Dependencies and installation
 
-## 2.2 Setup
+Dependencies and installation information are documented on the LORIS-MRI
+  [README.md](../README.md) file.
 
-## 2.3 Configuration
 
-## 2.4 Pipeline flow
+## 2.2 Configuration
 
-## 2.5 Common errors
+**NEED TO TALK ABOUT THE CREATION OF NIFTI OPTIONS AS IT IS REFERENCED IN THE
+  PIPELINE FLOW DOCUMENTATION.**
 
-[FAQ](AppendixA-FAQ.md)
+## 2.3 Pipeline flow
+
+By default the pipeline is designed for **raw DICOM MRI data**, collected by a
+  longitudinally-organized multisite study with a defined data acquisition
+  protocol. With modifications and further customization, it can handle any
+  modality of imaging data.
+
+Default images insertion into LORIS follows the following steps:
+
+1. Upload of a zip of the DICOM study via the **Imaging Uploader** module (or
+     DICOM study transfer directly on the server hosting LORIS).
```

Comment:
here we seem to allude again to the DICOM server direct transfer which would call dicomTar.pl and I thought we agreed to not stress this? maybe you mean either front end or back end transfer of the images